README.md

```diff
@@ -20,23 +20,136 @@ Required parameters:
 
 | Parameter        | Type  |  Description |
 | :-------------   | :---- | :-------------|
-| scmUrl        | String | Scm Url to format |
+| scmUrl        | String | Scm url to format |
```

Comment:
Removed `formatScmUrl` function since I don't think we'll need it anymore as we are switching to `scmUri`